src/core/io_queue.cc

```diff
@@ -211,6 +211,11 @@ class io_queue::priority_class_data {
     metrics::metric_groups metric_groups;
 };
 
+io_queue::stream::stream(shared_throttle& st, fair_queue::config cfg)
+        : fq(std::move(cfg))
+        , thr(st)
+{ }
+
 class io_desc_read_write final : public io_completion {
```

Comment:
Because of the removal of the need to notify about completion, this class does not need to keep `_stream` and ` fq_capacity` members.